_includes/get_pub_list.html

```diff
@@ -28,7 +28,7 @@
 
     {% assign fa_name = "Unknown" %}
     {% if pub.focus-area %}
-      {% assign fa_name = pub.focus-area %}
+      {% assign fa_name = pub.focus-area | join ' ' %}
```

Comment:
This change might not have been necessary, I was fixing something and tried this.

Reply:
I've dropped this, I think that's okay.